packages/quantification/src/quantification.ts

```diff
@@ -337,19 +336,18 @@ const getGrandfatherableYears = ({
 } => {
   const effectiveCurrentYear = quantifyAsOfYear ?? CURRENT_YEAR;
   const startYearIndex = Math.max(
-    modeledYears.findIndex(
-      (e) =>
-        e ===
-        subtract(effectiveCurrentYear, maxNumberGrandfatheredYearsForProject)
+    modeledYears.indexOf(
+      subtract(effectiveCurrentYear, maxNumberGrandfatheredYearsForProject)
     ),
     0
   );
-  const grandfatherableYears = modeledYears.slice(
-    startYearIndex,
-    modeledYears.indexOf(effectiveCurrentYear)
-  );
+  const endYearIndex = modeledYears.indexOf(effectiveCurrentYear);
+  const grandfatherableYears =
```

Comment:
This fixes the bug where when findIndex/indexOf can't find the index searched for, it returns -1. when -1 is the second argument to `.slice`, it returns slices to the second to last index. Instead, we want it to slice to the end of the array.